src/test/capabilities_test.ts

```diff
@@ -17,8 +17,7 @@ import * as capabilities from '../capabilities';
 
 function assertBrowserCapabilities(
     userAgent: string, expect: capabilities.BrowserCapability[]) {
-  assert.deepEqual(
-      capabilities.browserCapabilities(userAgent), new Set(expect));
+  assert.sameMembers([...capabilities.browserCapabilities(userAgent)], expect);
```

Comment:
I'm guessing the error reporting got updated since then, I get a nice message now:

1) capabilities chrome has all the capabilities:
     AssertionError: expected [ 'es2015', 'push', 'serviceworker' ] to have the same members as [ 'es2015', 'serviceworker' ]